src/main/java/de/hysky/skyblocker/config/configs/UIAndVisualsConfig.java

```diff
@@ -179,6 +179,8 @@ public static class OldBarPositions {
 
         @SerialEntry
         public OldBarPosition experienceBarPosition = OldBarPosition.LAYER1;
+        @SerialEntry
+        public OldBarPosition speedBarPosition = OldBarPosition.LAYER1;
 
```

Comment:
This is for legacy reasons, not needed here.
```suggestion
        public OldBarPosition experienceBarPosition = OldBarPosition.LAYER1;
```

Reply:
I think it's kinda needed? for default positioning, unless i don't remember my own code

Reply:
Woah I messed up. I will fix.